docs/source/concepts/PropertiesFile.rst

```diff
@@ -100,6 +100,14 @@ Directory Properties
 | ``defaultsave.directory``            | A default directory to use for saving files.      | ``../data``                         |
 |                                      | the data archive                                  |                                     |
 +--------------------------------------+---------------------------------------------------+-------------------------------------+
+| ``errorreports.core_dumps``          | *Linux only*                                      | ``/var/lib/apport/coredump``        |
+|                                      | Set this variable to your system's core file      |                                     |
+|                                      | location and the error reporter will automatically|                                     |
+|                                      | extract C++ stacktraces after a crash.            |                                     |
+|                                      | You may find this location listed in              |                                     |
+|                                      | ``/proc/sys/kernel/core_pattern`` (See            |                                     |
+|                                      | `here <kernel>`_ for more info).                  |                                     |
```

Comment:
The link to the kernel documentation doesn't work. It is trying to find it in the concepts folder where this PropertiesFile is located.